gqlalchemy/transformations/importing/loaders.py

```diff
@@ -616,6 +618,8 @@ def _save_row_as_relationship(
             property_from: Property of the source node.
             property_to: Property of the destination node.
             relation_label: Label for the relationship.
+            table_name: Name of the table used to read parameters
+            parameters: Relationship parameters to be added
```

Comment:
Again here, a bit of a different definition. It makes more sense to say "Relationship properties to be added". How did you decide on the `parameters` to begin with?